### DISCUSSION TITLE
Enchantment costs

### DISCUSSION THREAD

**Discussion by QuietFantasy** [Geolykt/EnchantmentsPlus#95] [Q&A] [answered]
Hello, was just wondering what the enchantment cost for the enchants are when going through an anvil like how many levels it costs, or what file to look through to find this info.

**Comment by Geolykt** [comment, marked as answer]
For the 1xx anvil merger, no additional level cost is created, i.e. the cost is as if the enchantments were not there. However in the 2xx anvil merger (default since 2.0.0) the cost in levels will be either 4 or 7 times the amounts of custom enchantments on the output item additional to the vanilla costs. In most cases it'll be 7, however when the items cannot be merged under vanilla circumstances it'll be 4. Why that? Don't ask, enchantment merging is pure pain. More info can be found in the <https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/src/main/java/de/geolykt/enchantments_plus/evt/NewAnvilMerger.java> source file.

**Reply by QuietFantasy** [reply]
thank you for the answer
